### ISSUE TITLE
How to directly test the program?

### ISSUE THREAD

**Issue by hanqim** [xindongzhang/ELAN#9]

**Comment by wxslby** [comment]
Have you solved it? I really want to know how to test and inference it.

**Comment by yongsongH** [comment]
I have the same problem. Looking forward to getting a response addressed this concern.

**Comment by Wedream-wj** [comment]
I try to code to test and inference it, but the results look abnormal.  I also want to know how to test and inference it.

**Comment by sunsunshark** [comment]
> I try to code to test and inference it, but the results look abnormal. I also want to know how to test and inference it.
Me too.The yellow tone in the results is very heavy.Have you get the right code ,friends?

**Comment by zhang13359** [comment]
> 我尝试编写代码来测试和推断它，但结果看起来不正常。我也想知道如何测试和推断它。



> 我也遇到了同样的问题。期待得到解决此问题的答复。



> 你解决了吗？我真的很想知道如何测试和推断它。

tensor = sr.squeeze(0)
tensor = tensor.detach().cpu().numpy().transpose(1, 2, 0)
image = cv2.imwrite(sr_file, tensor)
